### PR TITLE
Add support user_tags field in library to append volume labels

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

**Review by arahamad — APPROVED** [review]
lgtm, please verify these changes before merging as we are in process for compute identity release

**Comment by contsto** [comment]
:red_circle: Coverage decreased from [90.3926%] to [87.7868%]

**Comment by contsto** [comment]
:red_circle: Coverage decreased from [90.3926%] to [87.7868%]

**Comment by contsto** [comment]
:red_circle: Coverage decreased from [90.3926%] to [87.7868%]

File: block/provider/create_volume.go [hunk 1]
```diff
@@ -38,6 +38,10 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 	defer vpcs.Logger.Debug("Exit from CreateVolume method...")
 	defer metrics.UpdateDurationFromStart(vpcs.Logger, "CreateVolume", time.Now())
 
+	if len(vpcs.Config.VPCConfig.ClusterVolumeLabel) != 0 {
```
Comment (arahamad) [inline block/provider/create_volume.go:41]: can you write this in `validateVolumeRequest` method

File: go.mod [hunk 2]
```diff
@@ -4,7 +4,7 @@ go 1.15
 
 require (
 	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7
-	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta4
+	github.com/IBM/ibmcloud-volume-interface v1.0.1-beta4.0.20220620123330-2ce06f9f3405
```
Comment (arahamad) [inline go.mod:7]: update this with proper release

**Review by arahamad — CHANGES_REQUESTED** [review]

**Review by arahamad — APPROVED** [review]
lgtm, please verify also

**Comment by contsto** [comment]
:red_circle: Coverage decreased from [90.3926%] to [87.7868%]

**Review by ambiknai — APPROVED** [review]

**Pull request merged**